recipes/cygwin/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: cygwin
+  version: "2.4.1"    # this is the version of the cygwin dll.  Not sure how relevant it is.
+
+# don't download source here - seems there might be some redirect that leads to a different file.
```

Comment:
Does sound like a redirect problem, but wouldn't the right fix here be to determine how to address the redirection in `conda build`?